docs/source/openvino/inference.mdx

```diff
@@ -22,403 +25,158 @@ Once [your model was exported](export), you can load it by replacing the `AutoMo
+batch_size, num_images, height, width = 1, 1, 512, 512
+pipeline.reshape(batch_size=batch_size, height=height, width=width, num_images_per_prompt=num_images)
+images = pipeline(prompt, height=height, width=width, num_images_per_prompt=num_images).images
 ```
 
-|   |   |
-|---|---|
-| ![](https://huggingface.co/datasets/optimum/documentation-images/resolve/main/intel/openvino/textual_inversion/sdxl_without_textual_inversion.png) | ![](https://huggingface.co/datasets/optimum/documentation-images/resolve/main/intel/openvino/textual_inversion/sdxl_with_textual_inversion.png) |
+## Configuration
 
-
-#### Image-to-Image
-
-Here is an example of how you can load a PyTorch SDXL model, convert it to OpenVINO on-the-fly and run inference using OpenVINO Runtime for *image-to-image*:
+The `ov_config` parameter allow to provide custom OpenVINO configuration values. This can be used for example to enable full precision inference on devices where FP16 or BF16 inference precision is used by default.
 
 ```python
-from optimum.intel import OVStableDiffusionXLImg2ImgPipeline
-from diffusers.utils import load_image
-
-model_id = "stabilityai/stable-diffusion-xl-refiner-1.0"
-pipeline = OVStableDiffusionXLImg2ImgPipeline.from_pretrained(model_id, export=True)
-
-url = "https://huggingface.co/datasets/optimum/documentation-images/resolve/main/intel/openvino/sd_xl/castle_friedrich.png"
-image = load_image(url).convert("RGB")
-prompt = "medieval castle by Caspar David Friedrich"
-image = pipeline(prompt, image=image).images[0]
-# Don't forget to save your OpenVINO model so that you can load it without exporting it with `export=True`
-pipeline.save_pretrained("openvino-sd-xl-refiner-1.0")
+ov_config = {"INFERENCE_PRECISION_HINT":"f32"}
+model = OVModelForSequenceClassification.from_pretrained(model_id, ov_config=ov_config)
 ```
 
-
-#### Refining the image output
-
-The image can be refined by making use of a model like [stabilityai/stable-diffusion-xl-refiner-1.0](https://huggingface.co/stabilityai/stable-diffusion-xl-refiner-1.0). In this case, you only have to output the latents from the base model.
-
+Optimum Intel leverages OpenVINO's model caching to speed up model compiling on GPU. By default a `model_cache` directory is created in the model's directory in the [Hugging Face Hub cache](https://huggingface.co/docs/huggingface_hub/main/en/guides/manage-cache). To override this, use the ov_config parameter and set `CACHE_DIR` to a different value. To disable model caching on GPU, set `CACHE_DIR` to an empty string.
 
 ```python
-from optimum.intel import OVStableDiffusionXLImg2ImgPipeline
-
-model_id = "stabilityai/stable-diffusion-xl-refiner-1.0"
-refiner = OVStableDiffusionXLImg2ImgPipeline.from_pretrained(model_id, export=True)
-
-image = base(prompt=prompt, output_type="latent").images[0]
-image = refiner(prompt=prompt, image=image[None, :]).images[0]
+ov_config = {"PERFORMANCE_HINT": "LATENCY", "CACHE_DIR":""}
```

Comment:
```suggestion
ov_config = {"CACHE_DIR":""}
```
It used to be needed to specify LATENCY performance hint when changing any other ov_config value, but that is no longer the case (LATENCY is always the default), so the example is clearer without it.